### DISCUSSION TITLE
Single docker-compose.yaml file. How to migrate?

### DISCUSSION THREAD

**Discussion by thehijacker** [louislam/dockge#179] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Hello,

Up until now I was using single docker-compose.yaml file with over 30 services inside. Painfull to manage so I am thinking about trying dockge. But how to "migrate"/"import" this properly? I need to make a subfolder under stacks folder for every image that I am using or is there an import/migrate option somewhere that I am missing.

Thank you.

### 🐻 Dockge Version

1.1.1

### 💻 Operating System and Arch

Docker LXC in Proxmox

### 🌐 Browser

Google Chrome latest

### 🐋 Docker Version

24.0.7, build afdd53b

**Comment by trythatagain** [comment]
This is fairly easy in general, but I expect it all comes down to how you've written your 1 yml file. I've only really only done single services per compose.yml (unless a DB is needed). So if I can ask a few questions.
1. where do you store your config files for each service, I use something like (appdata/container-name) 
2. Where did you get most of your container configs from, I use github and lscr.io/linuxserver, if either of these then it's probably quite easy to do.

Just take each service config and click '+Compose' and enter the information as in the template within Dockge, make sure all paths are set correct.

It's easier to do than explain really, Dockge when you click '+Compose', fill in the compose config and click 'Save', Dockge will then create a folder named after your stack, and create the compose.yml file there.

Once you setup Dockge open the folder where your compose files will be created and click '+Compose' and create the example there, by naming the stack and saving, then have a look at your compose folder.

Like I say, it's easier to do than explain. But, if you need more info, please just ask
